helpers/io.go

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024-Present Defense Unicorn pkg Authors
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2024-Present Defense Unicorns
```

will have to change this globally xD